gateway/1.7/modules/ROOT/pages/policies-included-external-authorization.adoc

```diff
@@ -16,11 +16,19 @@ endif::[]
 
 == Summary
 
-The External Authorization policy forwards incoming client HTTP requests to an external gRPC of HTTP authorization service to determine if the client HTTP request is authorized to access the upstream service. In the request is authorized, Flex Gateway allows the request to pass to the upstream service. For unauthorized accesses, Flex Gateway returns a `403` forbidden request error. 
+The External Authorization policy forwards incoming client HTTP requests to an external gRPC of HTTP authorization service to determine if the client HTTP request is authorized to access the upstream service. If the request is authorized, Flex Gateway allows the request to pass to the upstream service. For unauthorized accesses, Flex Gateway returns a `403` forbidden request error. 
 
-To forward request headers, such as an authorization token header, to the authorization service, use the *Allowed Headers* parameter to configure which headers from the client HTTP requests to send to the authorization service.
+To forward request headers, such as an authorization token header, to the authorization service, use the *Allowed Headers* parameter to configure which headers from the client HTTP requests to send to the authorization service. For policies with an HTTP Authorization server, you can add headers not included in the client request to the authorization request by defining the headers in the *Headers to add* parameter.
```

Comment:
For policies with an HTTP Authorization server, you can add headers not included in the client request to the authorization request by defining the headers in the *Headers to add* parameter.

I guess this shouldn't be added, in grpc server the server can add the headers too. probably we should just remove it